client/src/components/NewParkReviewForm.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from "react"
+import translateServerErrors from "../services/translateServerErrors"
+import ErrorList from "./layout/ErrorList"
+
+const NewParkReviewForm = (props) => {
+    const parkId = props.parkId
+    const defaultReview = {
+        rating: "",
+        reviewText: ""
+    }
+    const [newReview, setNewReview] = useState(defaultReview)
+    const [errors, setErrors] = useState([])
+
+    const postNewParkReview = async () => {
+        try {
+            const response = await fetch(`/api/v1/parks/${parkId}`, {
+                method: "POST",
+                headers: new Headers({
+                    "Content-Type": "application/json"
+                }),
+                body: JSON.stringify( {review: newReview} )
+            })
+            const responseBody = await response.json()
+            props.setPark(responseBody.park)
```

Comment:
the server should be sending back the one new review that was created, not extra information. The Park and other reviews are already in state